src/AppShed/Extensions/SpreadsheetBundle/Service/GeoService.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+namespace AppShed\Extensions\SpreadsheetBundle\Service;
+
+use GuzzleHttp\Client;
+
+/**
+ * Description of GeoService
+ *
+ * @author Victor
+ */
+class GeoService {
+
+    public function getGeo($address)
+    {
+        $guzzle = new Client();
```

Comment:
This is good as a service, but i still want you to make the Client a service.

This is so that

1. you only make it once
2. you can set the url and default query params in the xml

[This file](https://github.com/mcfedr/youtubelivestreamsbundle/blob/master/src/Mcfedr/YouTube/LiveStreamsBundle/Resources/config/services.yml#L14) shows you

How to set the url:

    base_url:
        - "http://maps.google.com/maps/api/geocode/json

It also has the default query params

    defaults:
        query:
            sensor: false

Then you can see [where it is used](https://github.com/mcfedr/youtubelivestreamsbundle/blob/master/src/Mcfedr/YouTube/LiveStreamsBundle/Streams/YouTubeStreamsLoader.php#L74)

    $response = $this->client->get('', [
        'query' => [
            'address' => $address
        ]
    ]);

This means you *dont* have to urlencode yourself!